lib/ReactViews/Notification/terriaErrorNotification.tsx

```diff
@@ -0,0 +1,111 @@
+import i18next from "i18next";
+import { runInAction } from "mobx";
+import React from "react";
+import TerriaError from "../../Core/TerriaError";
+import ViewState from "../../ReactViewModels/ViewState";
+import Collapsible from "../Custom/Collapsible/Collapsible";
+import parseCustomMarkdownToReact from "../Custom/parseCustomMarkdownToReact";
+
+const Text: any = require("../../Styled/Text").default;
+const Spacing: any = require("../../Styled/Spacing").Spacing;
+const RawButton: any = require("../../Styled/Button").RawButton;
+const TextSpan: any = require("../../Styled/Text").TextSpan;
+const Box: any = require("../../Styled/Box").default;
+
+const ErrorsBox = (props: {
+  errors: (Error | TerriaError)[];
+  depth: number;
+}) => {
+  return (
+    <>
+      {props.errors.map(error => (
+        <Box
+          displayInlineBlock
+          css={{
+            paddingLeft: "7px",
```

Comment:
Always use multiples of 2px for spacing! It makes it easier to line things up consistently.

Reply:
done!